build.sh

```diff
@@ -3,7 +3,7 @@
 apt-get update
 apt-get install -y upx
 python -m pip install --upgrade pip setuptools wheel
-cd workspace/
+cd workspace/ || exit
```

Comment:
Just out of curiosity, why this was added?

Reply:
`task shell:check` pointed out that it's not certain that `cd workspace/` is a successful operation, so leaving it as it was could lead to errors down the line. This way `exit` uses the same output of `cd workspace/` as its code. A more detailed explanation can be found here: https://www.shellcheck.net/wiki/SC2164